week2/assignments/05_navbar_and_banner.html

```diff
@@ -27,32 +28,50 @@
           - give justify-content as space-around to have spaces around flex items
           - set background-color to #eee
         */
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-around;
+        align-items: center;
+        background-color: #eee;
       }
       .navbar a {
         /*
           - set color to some dark like #333
           - remove the underline
         */
+        color: #333;
+        text-decoration: none;
       }
       .navbar-nav {
         /*
           - make display to flex
           - set direction as row
           - remove the list style by setting it to none
         */
+        display: flex;
+        flex-direction: row;
+        list-style: none;
+        
+        ;
```

Comment:
@meena108  Some unnecessary semi-colon although it will not harm anything ;) But be careful